src/components/lib/smartblocks/SmartBlockSurvey.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <survey :survey="survey" />
+</template>
+<script>
+import * as SurveyVue from 'survey-vue'
+import 'survey-vue/survey.css'
+SurveyVue.StylesManager.applyTheme('default')
+const Survey = SurveyVue.Survey
```

Comment:
Explicit import is always better, also it helps tree shaking as well. Would you test the suggestion below?
```suggestion
import { StylesManager, Survey } as SurveyVue from 'survey-vue'
import 'survey-vue/survey.css'
SurveyVue.StylesManager.applyTheme('default')
```